test/tbb/test_multifunction_node.cpp

```diff
@@ -547,6 +589,12 @@ TEST_CASE("Lightweight testing"){
     lightweight_testing::test<tbb::flow::multifunction_node>(10);
 }
 
+//! Test excesption thrown in node with lightweight policy was rethrown by graph
+//! \brief \ref error_guessing
+TEST_CASE("Exception in lightweight node"){
+    test_exception_ligthweight_policy();
+}
```

Comment:
I believe this test needs to be moved into some common space (e.g. `namespace lightweight_testing`?) and parameterized by a node type (as it is done for `test_lightweight`), tests per each functional node is extended by this test case.